synoptic/src/synoptic/tests/units/JsonExporterTests.java

```diff
@@ -0,0 +1,510 @@
+package synoptic.tests.units;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.junit.Before;
+import org.junit.Test;
+
+import synoptic.main.AbstractMain;
+import synoptic.main.PerfumeMain;
+import synoptic.main.options.PerfumeOptions;
+import synoptic.model.PartitionGraph;
+import synoptic.model.export.DotExportFormatter;
+import synoptic.model.export.JsonExporter;
+import synoptic.model.export.types.EvBasedGraph;
+
+public class JsonExporterTests {
+
+    private PerfumeMain main = null;
+    private PartitionGraph pGraph = null;
+    private EvBasedGraph evGraph = null;
+    private Object expectedJSON = null;
+    private JSONObject expectedJSONObject = null;
+    private JSONParser parser = null;
+    private String dispJString = null;
```

Comment:
In general (but almost always), global variables should be avoided when possible. Almost all of these should not be global and should be private, method-level variables. For example, `main` is only used within `setUp()`. The only reason a global variable should be used is if the state of an object needs to be shared between methods (not just the variable being re-initialized with the same name in different methods but actually the same object) and if there is no other convenient way to pass that object between the methods (e.g., parameters, return values). Please make these local variables instead of global.